README.md

```diff
@@ -72,75 +72,108 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
-- System plugin - Small other changes to improve the usability of the Recycle Bin command. Thanks [@htcfreek](https://github.com/htcfreek)!
-- WindowWalker plugin - Show all open windows with action keyword. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Show Steam (steam://open/) shortcuts in the Program plugin. 
+- Localize paths of Program plugin results. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Support MSC and CPL files in "Run command" results of Program plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Added missing MSC and CPL settings to the results of Windows Settings plugin. Thanks [@htcfreek](https://github.com/htcfreek)!
+- System plugin: Setting for separate "Open/Empty Recycle bin" results or single result with context menu. Thanks [@htcfreek](https://github.com/htcfreek)!
+- Add support for log2 and log10 in Calculator plugin. Thanks [@RickLuiken](https://github.com/RickLuiken)!
+- Removed the TimeZone plugin.
+- Fix the crash when loading thumbnail for PDF files. (This was a hotfix for 0.67)
+
+### Shortcut Guide
+
+- Added: Dismiss Shortcut Guide with mouse click. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
 
 ### Quick Accent
 
-- Added dashes characters. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
-- Added Estonian characters. Thanks [@jovark](https://github.com/jovark)!
-- Added Hebrew characters. Thanks [@Evyatar-E](https://github.com/Evyatar-E)!
-- Added diacritical marks. Thanks [@Aaron-Junker](https://github.com/Aaron-Junker)!
-- Added Norwegian characters. Thanks [@norwayman22](https://github.com/norwayman22)!
+- Added Lithuanian characters. Thanks [@saulens22](https://github.com/saulens22)!
+- Added additional (Chinese) characters. Thanks [@ChaseKnowlden](https://github.com/ChaseKnowlden)!
+
 
 ### Settings
 
-- Fixed URL click crash on the "What's New" screen.
-- Added quick access system tray launcher. Thanks [@niels9001](https://github.com/niels9001)!
+- Add missing flyout borders on Windows 10. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Add experimentation settings landing page. Thanks [@chenss3](https://github.com/chenss3)!
+- Show icons of user-installed PowerToys Run plugins. Thanks [@al2me6](https://github.com/al2me6)!
+- Fixed crash when clicking Browse for backup and restore location while running elevated.
+- Respect taskbar position when showing system tray flyout. (This was a hotfix for 0.67)
+- Show correct Hosts module image. (This was a hotfix for 0.67)
 
 ### Documentation
 
-- Added PowerToys [disk usage footprint document](/doc/devdocs/disk-usage-footprint.md).
-- Fixed some grammar issues on main readme / Wiki.  Thanks [@CanePlayz](https://github.com/CanePlayz)!
+- Improve GPO docs and the layout of the disabled Experiments setting. Thanks [@htcfreek](https://github.com/htcfreek)!
```

Comment:
This was in internal fix after merging pr into dev. The wrong ui and docs was never released. Should we really mention this?

Reply:
good catch. Removing